ewatercycle/forcing/default.py

```diff
@@ -37,10 +39,15 @@ def save(self):
         """Export forcing data for later use."""
         yaml = YAML()
         yaml.register_class(self.__class__)
-        target = Path(self.directory) / 'ewatercycle_forcing.yaml'
-        # TODO remove directory or set to .
+        target = Path(self.directory) / FORCING_YAML
+        # We want to make the yaml and its parent moveable,
+        # so the directory should not be included in the yaml file
+        inner_file = StringIO()
+        yaml.dump(self, inner_file)
+        inner = inner_file.getvalue()
+        inner = inner.replace(f'directory: {self.directory}\n', '')
         with open(target, 'w') as f:
-            yaml.dump(self, f)
+            f.write(inner)
```

Comment:
This works fine, but I have a suggestion to refactor these lines:

```python
import copy 

        content = copy.copy(self)
        del content.directory
        with open(target, 'w') as f:
            yaml.dump(content, f)
```

What do you think? it is also possible to use deepcopy() instead of copy().

Reply:
Thanks, that is much more readable. 

The deepcopy() is not needed as the clone only gets a string attribute deleted and the shallow copy() already takes care of copying that.